packages/web-react/docs/stories/feature-flags/Overview.stories.mdx

```diff
@@ -0,0 +1,58 @@
+import { Meta } from '@storybook/blocks';
+
+<Meta title="Experimental/Feature Flags" />
+
+# Feature Flags
+
+## Table of Contents
+
+- [Current feature flags](#current-feature-flags)
+- [Turning on feature flags in SASS](#turning-on-feature-flags-in-sass)
+
+The `@lmc-eu/spirit-web-react` codebase ships with a number of feature flags. These feature
+flags enable new behavior and styling, allowing you to opt-in to new (and
+sometimes breaking) changes while using the current version. A feature flag may
+be configured in React, SASS, or both.
+
+This section in the Storybook documents each feature flag that is available and
+how to configure it. Folders and stories within this section in Storybook show
+components with all feature flags turned on.
+
+## Current feature flags
+
+Flags prefixed with `enable-*` contain features we'd like consuming projects to
+test and give us feedback on. They're generally stable and unlikely to change
+but may change based on your feedback.
+
+Flags prefixed with `enable-v#-*` will be marked as `true` or "on" by default in the next major version.
```

Comment:
Maybe somehow reword it. Because this is not what we do. We just remove the flag and make the behavior the default.

```suggestion
Flagged features prefixed with `enable-v#-*` will be made the default in the version specified in the name. 
```

Reply:
Fixed.